examples/3_Advanced_Topics/3-3_Database-specific_configuration.py

```diff
@@ -69,3 +69,17 @@
 spec_query = spec_query.with_batch_size(5)
 spec_query
 # -
+
+# ## Specification to Specification links
+
+# This package supports traversing specification to specification links, by default all such links will be followed.
+# For typical databases this is the correct and desired behaviour, however in some circumstances this may cause query
+# times and response sizes to become very large. In such cases you can control the number of nested links that will be
+# followed using the ``maximum_spec_link_depth`` parameter on the ``BomAnalyticsClient`` object.
```

Comment:
```suggestion
# using the ``maximum_spec_link_depth`` parameter on the ``BomAnalyticsClient`` object.
```

Reply:
Can you add a Restructured Text xref in here, to link to the actual property documentation?